zbus_macros/src/iface.rs

```diff
@@ -447,6 +441,7 @@ pub fn expand(args: Punctuated<Meta, Token![,]>, mut input: ItemImpl) -> syn::Re
             args_names,
             reply,
             member_name,
+            cfg_attrs,
```

Comment:
I appreciate your attention to details a lot here and also how you're keeping the commits atomic. :+1:  I also appreciate that you're adding the emojis but please use the appropriate one for each. E.g this isn't a bug fix but rather an internal refactor so ♻️ would be better.